core/core-load-paths.el

```diff
@@ -94,8 +94,8 @@
 ;;
 ;; This is based on `user-emacs-directory', not `spacemacs-start-directory',
 ;; because Spacemacs may be installed to a shared location and this directory
-;; and its children should be per-user.
-(defconst spacemacs-cache-directory
+;; and its children should be per-user. Allow customer define their own value.
+(defcustom spacemacs-cache-directory
   (concat user-emacs-directory ".cache/")
   "Spacemacs storage area for persistent files.")
```

Comment:
Please specify at least `:type` and `:group` (the latter may sometimes be implied by a `defgroup` in the same file, but there is none in this file).

Reply:
Added these two fields. Thanks.